### DISCUSSION TITLE
Page width

### DISCUSSION THREAD

**Discussion by lanbugs** [nunocoracao/blowfish#1536] [Q&A]
Hi,

is there a simple way to set the page on 100% width and use all the space?

Regards,

Max

**Comment by knowinginner** [comment]
### As theme is base on tailwind, 

the formats of width are as formulated in [tailwind css](https://tailwindcss.com/docs/width). For full width look for example at [recent articles cardview-fullwidth](https://github.com/nunocoracao/blowfish/blob/main/layouts/partials/recent-articles/cardview-fullwidth.html),

where definition is class="relative w-screen max-w-[1600px] px-[30px]" style="left: calc(max(-50vw,-800px) + 50%);"

Cheers

Vlad
